test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs

```diff
@@ -91,7 +92,12 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
             Assert.Contains(expectedCompilationContent, content);
         }
 
-        [Fact]
+        [ConditionalFact]
+        // MVC Generates portable PDBs when compiling Razor views. NET461 does not understand portable PDBs
```

Comment:
I reference this test in the MVC issue so once that closes out it should be a reminder to revert this.